Public/Src/IDE/Debugger/DScriptDebugerRenderer.cs

```diff
@@ -0,0 +1,270 @@
+﻿// Copyright (c) Microsoft. All rights reserved.

```

Comment:
This is pretty much DScript-specific stuff extracted from the old `Renderer.cs` into a separate file (i.e., no effective changes here)